calyx/frontend/src/workspace.rs

```diff
@@ -79,16 +80,33 @@ impl Workspace {
             return Ok(relative_import);
         }
 
-        let library_import = lib_path.join(import.clone());
-        if library_import.exists() {
-            return Ok(library_import);
+        let library_imports: Vec<_> = lib_paths
+            .iter()
+            .filter_map(|lib_path| {
+                let library_import = lib_path.join(import.clone());
+                library_import.exists().then_some(library_import)
+            })
+            .collect();
+        if library_imports.len() == 1 {
+            return Ok(library_imports.into_iter().next().unwrap());
```

Comment:
nit: since library imports is a vec, it would make more sense to just directly
index the zeroth entry

Reply:
my bad, missed the return type somehow! nvm